Sources/LiveViewNative/Modifiers/AspectRatioModifier.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AspectRatioModifier.swift
+// LiveViewNative
+//
+//  Created by May Matyi on 3/23/23.
+//
+
+import SwiftUI
+
+struct AspectRatioModifier: ViewModifier, Decodable {
+    private let aspectRatio: CGSize
+    private let contentMode: ContentMode
+
+    init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+
+        self.aspectRatio = try container.decode(CGSize.self, forKey: .aspectRatio)
```

Comment:
The aspect ratio should be optional, since it is in SwiftUI.

The body of the modifier will have to be slightly different, since the aspect ratio-less variant takes a `CGFloat?` not a `CGSize?`:

```swift
if let aspectRatio {
    content.aspectRatio(aspectRatio, contentMode: contentMode)
} else {
    content.aspectRatio(contentMode: contentMode)
}
```

Reply:
Oh, good catch. Fixed!